README.md

```diff
@@ -4,9 +4,13 @@
 
 > Protfolio2
 
-![screenshot](./Screenshot.png)!
+![screenshot](images/screenshot.png)!
```

Comment:
- [ ] The `screenshot` in the README file is still broken. I would highly recommend fixing that to make README professional.